features/core/Header.tsx

```diff
@@ -49,9 +49,7 @@ const Header = () => {
   return (
     <Box display="flex" justifyContent="space-between" alignItems="center">
       <Box>
-        <Typography variant="h4">
-          <i>EMP Tools</i>
-        </Typography>
+        <Typography variant="h4">⚒️⚡️EMP Tools</Typography>
```

Comment:
+9999

Reply:
@mrice32 @kendricktan @adrianmcli @daywiss Please suggest other emoji's here or forever hold your peace! (until the next PR).

Reply:
so you guys can see it more easily it looks like this now:
![image](https://user-images.githubusercontent.com/12886084/88365968-0452c100-cd88-11ea-8fdc-1a9652fcd7d1.png)